sdk/rust/oak/src/stubs.rs

```diff
@@ -0,0 +1,87 @@
+//
+// Copyright 2020 The Project Oak Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+//! Stub definitions of the ABI symbols, to allow Oak Node code to be built
+//! on the host platform.
+//!
+//! When building the Rust code for an Oak Node, the resulting library code
+//! references the functions provided by the Oak ABI.
+//!
+//! For the target Wasm platform, these Oak ABI functions are provided as Wasm
+//! host functions by the Oak Runtime; however, the Runtime is not available
+//! when compiling on the host platform.
+//!
+//! On some host platforms (e.g. `target_os = "linux"`), linking a library that
```

Comment:
This is quite annoying :(

Reply:
Yep.  It took a little bit of figuring out, so I thought a nice big comment would help for the future.